src/send_packets.c

```diff
@@ -498,11 +498,13 @@ send_packets(tcpreplay_t *ctx, pcap_t *pcap, int idx)
     bool now_is_now;
 
     ctx->skip_packets = 0;
-    start_us = TIMEVAL_TO_MICROSEC(&ctx->stats.start_time);
+    ctx->first_time = true;
+    gettimeofday(&now, NULL);
```

Comment:
This will cause regression issues because every packet is timestamped. Timestamping is hugely expensive and is only done when absolutely necessary.

Overall, the logic is that timestamps are to be done only when necessary, or if it is safe to do so. Timestamps are hugely expensive, especially at 10GigE + speeds. For example, if it is determined that TX is behind by the designated rate by 10msec on a 10GigE link, the loop will send 1.25MB with no timestamping. This usually allows TX to catch up.

This logic allows me to specify, say 9.5Gbps rate and I get 9.5Gbps. With this mod, I would expect that to drop to about 6-8Gbps.